### ISSUE TITLE
Make speaker bios collapsable-expandable

### ISSUE THREAD

**Issue by lakruzz** [Praqma/code-conf.com#169]
As described in #137 the speakers should be shown in collapsed form, at could be collapsapsable/expandable

![image](https://cloud.githubusercontent.com/assets/155492/15286562/1b957bfe-1b5e-11e6-8837-14bf554f8f15.png)
(see #137 for  details)

It important that if a uses browses to the speakers page with an anchor:

e.g `http://www.code-conf.com/speakers/#seb-rose`

then Seb Rose should be EXPANDED as default!

**Comment by illus0r** [comment]
@lakruzz @jonnycoull 
Are you sure we need this feature? Real data looks like this and I see no problems to solve with "read more" hack

**Comment by illus0r** [comment]
Scrolling is more easy than clicking

**Comment by jonnycoull** [comment]
On the copy I had originally was much longer!! But , on mobile it will be a
huge amount to scroll through. So that's why it's added. It's ok it the
screen is nice and wide with plenty space for copy, but that may not be the
case for most users. Plus it's better if most faces are seen on the page as
opens to see the volume of speakers and not to ask the user to discover
that for themselves.

**Comment by illus0r** [comment]
@jonnycoull 
Make sense! But we can implement this behavior on mobile only. Do we really need it on desktop?

**Comment by illus0r** [comment]
@jonnycoull what do you thing about implementation of this for mobile only?

**Comment by jonnycoull** [comment]
ok, @illus0r go for that. maybe when more are added it could do with a name list down the side, we'll have a look next week when I'm back.

**Comment by illus0r** [comment]
@jonnycoull good, so I'll make items to collapse on mobile only.

**Comment by jonnycoull** [comment]
@illus0r cool!

**Comment by illus0r** [comment]
I've spent 1 hour more than planned because found possibility to reuse existing code.
But no time left to show `read more` on mobile only, now it's on every device.
I had no time to write custom styles, so I've reused old ones. We can make another task to finish it of expand time budget of this one.

![image](https://cloud.githubusercontent.com/assets/797993/18476625/791a0d0c-79d3-11e6-94d1-9dd32ee0778f.png)

![image](https://cloud.githubusercontent.com/assets/797993/18476490/fbdb9c48-79d2-11e6-9392-a350ede14bb7.png)

![image](https://cloud.githubusercontent.com/assets/797993/18476504/09651c72-79d3-11e6-8a2c-721b10c6b013.png)
